cli/tests/pcluster/validators/test_cluster_validators.py

```diff
@@ -345,7 +346,10 @@ def test_custom_slurm_settings_include_file_only_validator(
         ),
         pytest.param(
             None,
-            "test.slurmdbd.host",
+            ExternalSlurmdbd(
+                host="test.slurmdbd.host",
+                port=6819,
```

Comment:
Maybe it's better to test with a non-default port to verify we are updating the value.

Reply:
I'm not able to quickly set up the test we discussed about. Merging as it is.